Assets/VRTK/Scripts/Controls/2D/PanelMenu/PanelMenuController.cs

```diff
@@ -106,7 +106,7 @@ protected virtual void Update()
             {
                 if (rotateTowards == null)
                 {
-                    rotateTowards = GameObject.Find("Camera (eye)");
+                    rotateTowards = VRTK_SDK_Bridge.GetHeadsetCamera().gameObject;
```

Comment:
This still should be going through the DeviceFinder. replace with `VRTK_DeviceFinder.HeadsetCamera().gameObject`